c/clang_check.sh

```diff
@@ -22,13 +22,8 @@ RETURN=0
 for FILE in `ls *.[ch] */*.[ch]`; do
   DIFF=`diff $FILE <($CLANG_FORMAT $FILE)`
   if [ $? -ne 0 ]; then
-    if [ -z "$TRAVIS" ]; then
```

Comment:
The original intention is to format the files if running locally, but to fail the CI if there are differences.

Removing this will format the file in the CI test, but not commit it, and the CI will succeed.

I suggest replacing `$TRAVIS` with `$GITHUB_WORKFLOW`, here and below, since that [should be set](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables) in the CI run.